README.md

```diff
@@ -2,14 +2,20 @@
 
 This plugin allows applications that rely on Train to communicate with the WinRM API.  For example, you could use this to audit Windows Server 2016 machines.
 
-TODO - details about underlying library
+This plugin relies on the `winrm` and `winrm-fs` gems for implementation.
 
 Train itself has no CLI, nor a sophisticated test harness.  Chef InSpec does have such facilities, so installing Train plugins will require a Chef InSpec installation.  You do not need to use or understand Chef InSpec.
 
 Train plugins may be developed without a Chef InSpec installation.
 
 ## To Install this as a User
 
+### ChefDK Installation
+
+After June 2019, this plugin will be distributed with ChefDK; you do not need to install it separately.
+
+### Manual Installation
+
 Train plugins are distributed as gems.  You may choose to manage the gem yourself, but if you are an Chef InSpec user, Chef InSpec can handle it for you.
 
 You will need Chef InSpec v2.3 or later.
```

Comment:
Are these the beginning of instructions for a certain scenario? I'm uncertain if this section should be labeled under manual installation. Would Chef InSpec handling the management of the gem for the user be considered a manual installation?

Reply:
I have clarified the section title